Doc/library/pathlib.rst

```diff
@@ -1539,50 +1539,33 @@ Creating files and directories
 Copying, renaming and deleting
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-.. method:: Path.copy(target, *, follow_symlinks=True, preserve_metadata=False)
+.. method:: Path.copy(target, *, follow_symlinks=True, \
+                      preserve_metadata=False, dirs_exist_ok=False, \
+                      ignore=None, on_error=None)
 
-   Copy the contents of this file to the *target* file. If *target* specifies
-   a file that already exists, it will be replaced.
+   Copy this file or directory tree to the given *target*. If both paths are
+   existing files, the target file will be replaced.
 
-   If *follow_symlinks* is false, and this file is a symbolic link, *target*
-   will be created as a symbolic link. If *follow_symlinks* is true and this
-   file is a symbolic link, *target* will be a copy of the symlink target.
+   If a symlink is encountered and *follow_symlinks* is true (the default),
```

Comment:
This could be clearer for when copying a symlink, maybe "For a symlink if *follow_symlinks* is true..."?